src-electron/db/db-mapping.js

```diff
@@ -448,6 +448,8 @@ exports.map = {
       endpointVersion: x.DEVICE_VERSION, // Left for backwards compatibility
       deviceVersion: x.DEVICE_VERSION,
       deviceIdentifier: x.DEVICE_IDENTIFIER,
+      parentRef: x.PARENT_ENDPOINT_REF,
+      parentEndpointIdentifier: x.PARENT_ENDPOINT_IDENTIFIER,
```

Comment:
As an API consumer...  I assume parentEndpointIdentifier is the endpoint id visible in the spec?  What is parentRef?  Where is any of this documented?

Reply:
parentRef is the ENDPOINT_ID of the parent endpoint and parentEndpointIdentifier is the ENDPOINT_IDENTIFIER meaning the "endpoint id visible in the spec" as you mention. parentRef is important for things like checking if an endpoint with identifier x exists in the db...

I will add comments and a documentation page, thank you